### DISCUSSION TITLE
Allow button links to set their target

### DISCUSSION THREAD

**Discussion by adambeer** [directus/directus#11826] [Feature Requests]
It seems by default that the target of the anchor tag is always set to _blank which will open the link in a new window. There may be cases (an oauth flow) where you want to send the user away in the same window and then back to directus right after to hit an endpoint extension.

I looked around briefly in the presentation-links interface folder and couldnt see a reference to where the target was being set.

**Comment by rijkvanzanten** [comment]
Are you talking about the usage of the presentation-links interface specifically, or the more generic `v-button` component that's used internally / in extensions?

**Comment by adambeer** [comment]
The interface specifically. I see that v-buttons set their target to blank if the component is an anchor:

`:target="component === 'a' ? '_blank' : undefined"`

But what I am suggesting is that presentation-links should have the ability to override this behavior and maybe give the option for a _blank target or no value.

**Reply by rijkvanzanten** [reply]
thanks for confirming that 🙂

**Comment by scolastico** [comment]
I would love to see that feature

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
